cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Errors.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2017-2018, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.conversion;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public final class Errors {
```

Comment:
I think `Warnings` would be a more accurate name than `Errors`

Reply:
Agree, changed